site/global-guides/voice/authentication.mdx

```diff
@@ -0,0 +1,67 @@
+---
+id: inAppAuthGuide
+title: Authentication for Global In-App Calling
+slug: /voice/guides/inAppCalling/authentication
+description: Authentication In-App Calling (WebRTC)
+keywords:
+  - bandwidth
+  - voice
+hide_title: false
+image: '@site/static/img/bw-icon.svg'
+---
+## Retrieving an access token
+
+:::caution
+
+The token should be retrieved from a server running in a secure environment and securely provided to clients. Client-side javascript does not have a mechanism for hiding these credentials so **DO NOT** place these directly in your client-side code.
+
+Bandwidth accepts no responsibility for the loss of account credentials and any resulting network traffic, fraud, or undesired account access that results from failing to manage account access credentials in a completely secure manner.
+:::
+
+
+The process to retrieve an access token requires you to have your [WebRTC Credentials](./inAppGuide.mdx#generating-credentials-for-an-access-token) (ie. ClientID and ClientSecret), generated from the [Global Portal](https://login.voxbone.com/login). You will then be able to make a call to our token endpoint with these credentials.
+
+```mermaid
+sequenceDiagram
+    Application->>In-App Calling AuthServer: GET /api/v1/oauth2/token?scope=voice.webrtc <br> Headers: { Authorization: Basic PEJhc2ljIEF1dGggVXNlcm5hbWU+OjxCYXNpYyBBdXRoIFBhc3N3b3JkPg== }
+    In-App Calling AuthServer->>In-App Calling AuthServer: Validate credentials
+    In-App Calling AuthServer->>Application: 200 OK <br> {"access_token": "...", "access_token_id": "...", "token_type": "Bearer", "expires_in": 600}
+```
+### Request
+In order to retrieve the token, you will use Basic Authentication `ClientID:ClientSecret`. Below is a sample request:
+
+```sh
+curl --request POST \
+  --url https://id.voxbone.com/api/v1/oauth2/token?scope=voice.webrtc \
+  --header 'Authorization: Basic PEJhc2ljIEF1dGggVXNlcm5hbWU+OjxCYXNpYyBBdXRoIFBhc3N3b3JkPg==' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data grant_type=client_credentials
+```
+Now that we've shown the request, let's break down the parts of the API call.
+
+- **url** `https://id.voxbone.com/api/v1/oauth2/token?scope=voice.webrtc` - This is the In-App Calling AuthServers's `token_endpoint`. **IMPORTANT**: There is a query parameter of `scope`. This is used to limit the permissions in the token. When you request a token with the scope parameter that token will **ONLY** be able to access resources that require the specifiied scopes. In our example, we are limiting the token's permissions to only the WebRTC Gateway. If you do not provide a scope, then the token will be minted for **ALL** scopes that are allowed by the credentials when they were created.
+- **Headers**
+  - **Authorization** - This request requires authentication. You **MUST** provide your `ClientID` and `ClientSecret` in a Basic Authorization header. Basic Authentication uses a Base64 encoded string that contains the username and password.
```

Comment:
```suggestion
  - **Authorization** - Authentication is required. You **MUST** provide your `ClientID` and `ClientSecret` through a Base64 encoded Basic Authorization header. The encoded value should take the form `<ClientID>:<ClientSecret>`.
```

Reply:
My original comment changed this wording, but mainly I did not want the term "Basic Authorization header". Although, it's mostly understood what that is, the term "Basic Authentication" is the standard, not "Basic Authorization". The `Basic` is the authentication scheme.

Also, since the usage of `Base64-encoded` is an adjective, it requires a hyphen.

Reply:
```suggestion
  - **Authorization** - Authentication is required. You **MUST** provide your `ClientID` and `ClientSecret` using a Base64-encoded header with the `Basic` authentication scheme. The header value before encoding should take the form `Basic <ClientID>:<ClientSecret>`.
```
This is what it looks like if @matthewkmartin's and my suggestions are merged.